profiles/util_test.py

```diff
@@ -145,3 +150,54 @@ def test_make_thumbnail(self):
             mocked.assert_called_with(width, height, thumbnail_size)
             assert thumb_image.width == thumb_width
             assert thumb_image.height == thumb_height
+
+
+@pytest.mark.django_db
```

Comment:
I read [here](http://pytest-django.readthedocs.io/en/latest/helpers.html) that the reason is that the test needs the database, so the solution is to use the django `TestCase` and not the unittest one.